features/swap/components/TransactionTips.tsx

```diff
@@ -112,8 +133,44 @@ export const TransactionTips = ({
           </Text>
         )}
       </StyledDivForRateWrapper>
-
-      <Text variant="legend">${formattedDollarValue}</Text>
+      <Inline justifyContent={'flex-end'} gap={8} css={{ cursor: 'pointer' }}>
+        {tokenB?.tokenSymbol && (
+          <Tooltip label="Price impact due to the amount of liquity available in the pool.">
+            <Inline
+              css={{
+                backgroundColor: errorThreshold
+                  ? '$backgroundColors$error'
+                  : '$backgroundColors$secondary',
+                padding: '$2 $4 $2 $2',
+                borderRadius: 4,
+              }}
+            >
+              <ArrowUpIcon
+                color={errorThreshold ? 'error' : 'secondary'}
+                css={{
+                  transform: priceImpact < 0 ? 'rotate(180deg)' : undefined,
+                }}
+                size="medium"
+              />
+              <Text
+                variant="legend"
+                wrap={false}
+                css={{
```

Comment:
could be nice if there was a `color` prop for `Text` in junoblocks, this is good for this PR though 👍